### PR TITLE
created overview of third-party libs

### PR DESCRIPTION
**Pull request by weathersboyle**

### CODE REVIEW

File: android/third_party_libs.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Our Favorite Third-Party Libraries
+
+### Retrofit
+
+Clean, intuitive, RESTful HTTP client we use to perform network IO; use with OkHttp.
```
Comment (gdaniels) [inline android/third_party_libs.md:5]: Should probably link to OkHttp as well?

**Comment by gdaniels** [comment]
Aside from that one comment LGTM! :shipit: :one:

File: android/third_party_libs.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+Overview of OkHttp: <http://square.github.io/okhttp> <br/>
+Source for OkHttp: <https://github.com/square/okhttp>
+
+### Otto
+
+An event bus tailored to Android, Otto supports communication between distinct application components, when the use of a traditional interface is inadequate or cumbersome. 
+
+Overview: <http://square.github.io/otto> <br/>
+Source: <https://github.com/square/otto>
```
Comment (f2rf2r) [inline android/third_party_libs.md]: We should also add Greenrobot's EventBus and RxBus as alternatives

**Comment by guyIntrepid** [comment]
mmhmm :shipit:

**Comment by f2rf2r** [comment]
:ship: :shipit:

**Pull request merged**